concordium-sdk/src/main/java/com/concordium/sdk/responses/poolstatus/CurrentPaydayStatus.java

```diff
@@ -0,0 +1,48 @@
+package com.concordium.sdk.responses.poolstatus;
+
+import com.concordium.sdk.transactions.CCDAmount;
+import com.concordium.sdk.types.UInt64;
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+@Data
+@ToString
+@Getter
+public class CurrentPaydayStatus {
```

Comment:
Make fields final please.
I don't see the need for @Getter @ToString when @Data is used.

Reply:
done